### ISSUE TITLE
Buttons within drawer sidebar still visible using keyboard

### ISSUE THREAD

**Issue by siriwatknp** [siriwatknp/tailwindcss-jun-layout#5]
Not true:

visibility: hidden does allow the element to maintain its space in the layout (unlike display: none), but it does affect keyboard navigation in the following ways:

Hidden elements are not included in the tab order
Screen readers will not detect or announce these elements
The element cannot receive keyboard focus
